package.json

```diff
@@ -31,8 +31,8 @@
 		"CODE_OF_CONDUCT.md"
 	],
 	"scripts": {
-		"build": "tsc --project tsconfig.build.json",
-		"build:watch": "npm run build -- --watch --incremental",
+		"build": "cp -r core/static dist/core/static && tsc --project tsconfig.build.json",
```

Comment:
This should not be needed if we revert the `__dirname` change to using `PACKAGE_ROOT` again.